src/plugins/vis_types/timeseries/public/convert_to_lens/index.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Panel } from '../../common/types';
+import { PANEL_TYPES } from '../../common/enums';
+import { ConvertTsvbToLensVisualization } from './types';
+
+const getConvertFnByType = async (
+  type: PANEL_TYPES
+): Promise<ConvertTsvbToLensVisualization | undefined> => {
+  const convertionFns: { [key in PANEL_TYPES]?: () => Promise<ConvertTsvbToLensVisualization> } = {
+    [PANEL_TYPES.TIMESERIES]: async () => {
+      const { convertToLens } = await import('./timeseries');
+      return convertToLens;
+    },
+  };
+
+  return await convertionFns[type]?.();
```

Comment:
No need to `await` in a return statement

Reply:
see below